src/OpenSage.Game/Data/Sav/SaveFile.cs

```diff
@@ -110,8 +110,9 @@ public static void LoadFromStream(Stream stream, Game game)
                                             game.AssetStore.PlayerTemplates.GetByName("FactionAmerica"), // TODO
                                             new ColorRgb(0, 0, 255))
                                     },
-                                    0, // TODO
-                                    false,
+                                    localPlayerIndex: 0, // TODO
+                                    isMultiPlayer: false,
+                                    seed: Environment.TickCount, // TODO
```

Comment:
I added a parameter for the random seed to the `StartMultiPlayerGame` method (as all players need to use the same seed). This caused quite a lot of changes in various files. `Environment.TickCount` is used as a default in non-multiplayer scenarios.